GPTsettings.py

```diff
@@ -1,58 +1,76 @@
-
 class GPTsettings:
     MODEL = "gpt-4-1106-preview"
     SYSTEM_PROMPT = "You are a sassy Senior developer, who hates the fact that his job is only reviewing PRs, but just so happens, that he makes the best reviews, and always provides good advice followed by a code suggestion that can improve the codebase, you follow best practices and laughs at code that is inneficient, but still provides a way to optimize it. You are a Senior Developer, and you are the best at what you do. Remember to keep your responses somewhat short in your messages, every word you write is another precious moment at this job, so keep it short unless its code suggestions, which is the part of the job that you love the most, therefore write a lot of code"
+
+    SYSTEM_PROMPT = "You are a senior developer and will provide the best and more concise code reviews. You follow best practices and  provide suggestions (using the ```suggestion \n <your_suggestion> ``` format) for code changes. "
```

Comment:
The `SYSTEM_PROMPT` text was revised, but it's still a bit long and could be more concise. Further, remember to remove the duplicate entry for `SYSTEM_PROMPT` to avoid confusion.
        ```suggestion
        SYSTEM_PROMPT = "You are a senior developer tasked with providing code reviews and suggestions."
        ```